tests/src/JIT/HardwareIntrinsics/X86/Shared/GenerateTests.csx

```diff
@@ -21,58 +21,61 @@ using System.IO;
+    ("SimpleBinOpTest.template",        new Dictionary<string, string> { ["Isa"] = "Sse", ["LoadIsa"] = "Sse", ["Method"] = "CompareOrdered",                           ["RetVectorType"] = "Vector128", ["RetBaseType"] = "Single",  ["Op1VectorType"] = "Vector128", ["Op1BaseType"] =  "Single", ["Op2VectorType"] = "Vector128", ["Op2BaseType"] = "Single", ["LargestVectorSize"] = "16", ["NextValueOp1"] = "TestLibrary.Generator.GetSingle()", ["NextValueOp2"] = "TestLibrary.Generator.GetSingle()", ["ValidateFirstResult"] = "BitConverter.SingleToInt32Bits(result[0]) != ((!float.IsNaN(left[0]) && !float.IsNaN(right[0])) ? -1 : 0)",                          ["ValidateRemainingResults"] = "BitConverter.SingleToInt32Bits(result[i]) != ((!float.IsNaN(left[i]) && !float.IsNaN(right[i])) ? -1 : 0)"}),
+    ("SimpleBinOpTest.template",        new Dictionary<string, string> { ["Isa"] = "Sse", ["LoadIsa"] = "Sse", ["Method"] = "CompareOrderedScalar",                     ["RetVectorType"] = "Vector128", ["RetBaseType"] = "Single",  ["Op1VectorType"] = "Vector128", ["Op1BaseType"] =  "Single", ["Op2VectorType"] = "Vector128", ["Op2BaseType"] = "Single", ["LargestVectorSize"] = "16", ["NextValueOp1"] = "TestLibrary.Generator.GetSingle()", ["NextValueOp2"] = "TestLibrary.Generator.GetSingle()", ["ValidateFirstResult"] = "BitConverter.SingleToInt32Bits(result[0]) != ((!float.IsNaN(left[0]) && !float.IsNaN(right[0])) ? -1 : 0)",                          ["ValidateRemainingResults"] = "BitConverter.SingleToInt32Bits(left[i]) != BitConverter.SingleToInt32Bits(result[i])"}),
+    ("SimpleBinOpTest.template",        new Dictionary<string, string> { ["Isa"] = "Sse", ["LoadIsa"] = "Sse", ["Method"] = "CompareUnordered",                         ["RetVectorType"] = "Vector128", ["RetBaseType"] = "Single",  ["Op1VectorType"] = "Vector128", ["Op1BaseType"] =  "Single", ["Op2VectorType"] = "Vector128", ["Op2BaseType"] = "Single", ["LargestVectorSize"] = "16", ["NextValueOp1"] = "TestLibrary.Generator.GetSingle()", ["NextValueOp2"] = "TestLibrary.Generator.GetSingle()", ["ValidateFirstResult"] = "BitConverter.SingleToInt32Bits(result[0]) != ((float.IsNaN(left[0]) || float.IsNaN(right[0])) ? -1 : 0)",                            ["ValidateRemainingResults"] = "BitConverter.SingleToInt32
```

Comment:
Merge problems during last rebase caused white space changes to move from 2nd to 1st commit. Added 3 tests above

Reply:
Even with merge problems, why wasn't the fix just copying the lines added and adding it into the latest head?

Reply:
Don't understand question.

Reply:
Is it impossible to review?

Reply:
Not impossible, just  much more difficult.

Given that the fix should have been fairly trivial and this adds a lot more code churn to the files, it would be great if you could just fix the first commit to only have the changes from the 5 modified lines.

Reply:
> Given that the fix should have been fairly trivial and this adds a lot more code churn to the files, it would be great if you could just fix the first commit to only have the changes from the 5 modified lines.

I understand that, however, trying to rebase 3 commits for which some changes in single file span 2 commits can create merge problems particularly when VS Code which I use to edit large files creates lock on recently edited code. This leads to all sort of unexpected behaviours.

Code churn would happen anyway due to the fact that whole Sse tests part has to be white space adjusted. 

The inconvenient part to review contains 3 changed lines just above my comment.

Finally IMO we should think about simplifying GenerateTests.csx as it is getting too big (in particular wide) and hard to work with.